src/instruction.rs

```diff
@@ -37,7 +36,7 @@ impl Instructions {
 impl Drop for Instructions {
     fn drop(&mut self) {
         unsafe {
-            cs_free(self.ptr, self.len as libc::size_t);
+            cs_free(self.ptr, self.len as usize);
```

Comment:
why did this change?

Reply:
To eliminate the libc crate dependency

Reply:
I see.  Is that wise / portable though?

Reply:
I think in this case, it's okay. We only used simple `libc` types that have well-defined counterparts in Rust (like `size_t`, `usize`, `u8`) or are already defined in `std::os::raw` (like `c_uint`).

If we needed to use more complicated C types (such as structs) or call `libc` functions, then I would agree that we should use the `libc` crate.

Reply:
I'm not sure that `u8` is easily translateable from C, e.g., most infamously, the `char` type, but if we're just using `size_t` in FFI then I guess it's ok?

Also not sure what benefit removing the dep precisely is, since ostensibly, `capstone-sys` will have libc as a dep (and expose types from it), and hence transitively, so will this.

Unless `libc` was also removed as dep in `capstone-sys`?

Reply:
Capstone uses `uint8_t` (instead of char), so our use of `u8` is fine. Like you pointed out, it's a similar story for `size_t`/`usize`.

In the default configuration, `capstone-sys` has no dependencies, so normally `libc` is not in the dependency graph. 

Even when non-default features are used such that `libc` is a transitive dependency, it's nice to not have update the `libc` version in `Cargo.toml`.  =)